File: JavaScript_Basic_04_2023_SolutionsOfProblems/02.Ex_PB-JS-Conditional-Statements/P05_GodzillaVsKong.js
function godzillaVsKong(input) {
    let filmBudget = Number(input[0]);
    let actorNumber = Number(input[1]);
    let wearCost = Number(input[2]);
    let filmCost = actorNumber * wearCost;
    if (actorNumber > 150) {
        filmCost = filmCost * 0.9;
    }
    filmCost = filmCost + filmBudget * 0.1;
    if (filmCost > filmBudget) {
        console.log("Not enough money!");
        console.log(`Wingard needs ${(filmCost - filmBudget).toFixed(2)} leva more.`);
    } else {
        console.log("Action!");
        console.log(`Wingard starts filming with ${(filmBudget - filmCost).toFixed(2)} leva left.`)
    }
}

godzillaVsKong(["9587.88",
    "222",
    "55.68"])

File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P02Ex_ConditionalStatements/P05_GodzillaVsKong.java
package P02Ex_ConditionalStatements;

import java.util.Scanner;

public class P05_GodzillaVsKong {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double budget = Double.parseDouble(scanner.nextLine());
        int people = Integer.parseInt(scanner.nextLine());
        double priceOfClothes = Double.parseDouble(scanner.nextLine());

        if (people > 150)
            priceOfClothes = priceOfClothes * people * 0.9;
        else
            priceOfClothes = priceOfClothes * people;

        double totalPrice = priceOfClothes + budget * 0.1;

        if (totalPrice > budget) {
            System.out.println("Not enough money!");
            System.out.printf("Wingard needs %.2f leva more.", totalPrice - budget);
        } else {
            System.out.println("Action!");
            System.out.printf("Wingard starts filming with %.2f leva left.", budget - totalPrice);
        }
    }
}
